evaluator/evaluate_test.go

```diff
@@ -79,6 +80,35 @@ func TestEvaluate(t *testing.T) {
 	assert.Equal(t, 2, len(mockWriter.Messages))
 }
 
+func TestEvaluatePruneUpdates(t *testing.T) {
+	assert.NoError(t, os.Setenv("ENABLE_UPDATES_LATEST_ONLY", "true"))
+	TestEvaluate(t)
```

Comment:
There are two tests included in that one. Please split it to have two test methods.

Reply:
Fixed